### PR TITLE
10.3.0 Release

### PR DESCRIPTION
**Pull request by emilyrohrbough**

### CODE REVIEW

File: content/_changelogs/10.2.1.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  [#17853](https://github.com/cypress-io/cypress/issues/17853) and
+  [#22462](https://github.com/cypress-io/cypress/issues/22462).
+- Updated cross-origin cookie handling to align with browser behavior. This
+  fixed various issues with cookies when testing across multiple origins. Fixed
+  [#21363](https://github.com/cypress-io/cypress/issues/21363) and
+  [#20685](https://github.com/cypress-io/cypress/issues/20685).
+- Fixed a 9.x to 10.x migration bug where the migration tool would hang without
+  a response when it attempted to read the `pluginsFile` contents when the
+  `pluginsFile` value was a directory path instead of a relative file path.
+  Fixed
```
Comment (mjhenkes) [inline content/_changelogs/10.2.1.md:18]: need an issue here

File: content/_changelogs/10.2.1.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  `pluginsFile` value was a directory path instead of a relative file path.
+  Fixed [#22461](https://github.com/cypress-io/cypress/issues/22461).
+- Fixed the process profiler to output Cypress's summarized CPU and memory
+  metrics when the `cypress:server:util:process_profiler` debug stream is
+  enabled. Fixed [#22022](https://github.com/cypress-io/cypress/issues/22022).
+- Added a new clear button to easily clear the search text when filtering specs
+  in the Cypress runner. Fixes
+  [#21686](https://github.com/cypress-io/cypress/issues/21686).
+- Fix styling issue observed when long texted was entered in the search when
+  filter specs in the Cypress runner. Fixes
```
Comment (rockhold) [inline content/_changelogs/10.2.1.md:26]: Line 25: ```texted``` -> ```text```
Line 26: ```filter``` -> ```filtering```

Comment (emilyrohrbough) [inline content/_changelogs/10.2.1.md:26]:
```suggestion
- Fix styling issue observed when long text was entered in the search when
  filtering specs in the Cypress runner. Fixes
```

**Comment by emilyrohrbough** [comment]
This has been updated from 10.2.1 to 10.3.0.

File: content/_changelogs/10.3.0.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+- Fixed a memory leak in Chromium-based browsers caused by storing `data:` urls
+  containing base64 encoded files unnecessarily. Fixed
+  [#17853](https://github.com/cypress-io/cypress/issues/17853) and
+  [#22462](https://github.com/cypress-io/cypress/issues/22462).
+- Updated cross-origin cookie handling to align with browser behavior. This
+  fixed various issues with cookies when testing across multiple origins. Fixed
+  [#21363](https://github.com/cypress-io/cypress/issues/21363) and
+  [#20685](https://github.com/cypress-io/cypress/issues/20685).
+- Make an update to use posix paths to fix an issue where Cypress failed on
```
Comment (mjhenkes) [inline content/_changelogs/10.3.0.md:26]:
```suggestion
- Updated Cypress to use posix paths to fix an issue on
  Windows when a project required `node-ts`. Fixes
```

File: content/_changelogs/10.3.0.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+  containing base64 encoded files unnecessarily. Fixed
+  [#17853](https://github.com/cypress-io/cypress/issues/17853) and
+  [#22462](https://github.com/cypress-io/cypress/issues/22462).
+- Updated cross-origin cookie handling to align with browser behavior. This
+  fixed various issues with cookies when testing across multiple origins. Fixed
+  [#21363](https://github.com/cypress-io/cypress/issues/21363) and
+  [#20685](https://github.com/cypress-io/cypress/issues/20685).
+- Updated Cypress to use posix paths to fix an issue on
+  Windows when a project required `node-ts`. Fixes
+  Windows when a project required `node-ts`. Fixes
```
Comment (mjhenkes) [inline content/_changelogs/10.3.0.md:28]:
```suggestion
```
my bad

**Review by mjhenkes — APPROVED** [review]

**Review by pstakoun — APPROVED** [review]

**Pull request merged**